### PR TITLE
Fixed recipes containing GT arrows to use Thaumcraft elemental ones instead

### PR DESCRIPTION
**Pull request by dagger8243**
Fixed Recipes:

- Robes of the Stratosphere
- Wand Focus: Explosion
- Wand Focus: Shadowbeam

### CODE REVIEW

**Comment by dagger8243** [comment]
this what u mean caedis ? 
image[
](https://discord.com/channels/181078474394566657/603348502637969419/1258562736334442647)

**Comment by Caedis** [comment]
Please also include any in game screenshots of the recipe

File: src/main/java/com/dreammaster/scripts/ScriptEMT.java [hunk 1]
```diff
@@ -2709,25 +2709,12 @@ public void loadRecipes() {
...
+                        getModItem(Thaumcraft.ID, "ItemResource", 1, 1, missing), // Nitor
                         getModItem(Minecraft.ID, "firework_charge", 1, 0, missing),
-                        createItemStack(
-                                GregTech.ID,
-                                "gt.metaitem.02",
-                                1,
-                                26541,
-                                "{ench:[0:{lvl:3s,id:20s}],GT.HasBeenUpdated:1b}",
-                                missing),
-                        getModItem(Minecraft.ID, "tnt", 1, 0, missing),
```
Comment (chochem) [inline src/main/java/com/dreammaster/scripts/ScriptEMT.java:2721]: did you remove the tnt by mistake?

File: src/main/java/com/dreammaster/scripts/ScriptThaumicTinkerer.java [hunk 2]
```diff
@@ -2082,23 +2070,8 @@ public void loadRecipes() {
...
-                        createItemStack(
-                                GregTech.ID,
-                                "gt.metaitem.02",
-                                1,
-                                25330,
-                                "{ench:[0:{lvl:2s,id:21s}],GT.HasBeenUpdated:1b}",
-                                missing),
-                        GT_OreDictUnificator.get(OrePrefixes.gemExquisite, Materials.Diamond, 1L),
-                        GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Ichorium, 1L),
-                        getModItem(Minecraft.ID, "ghast_tear", 1, 0, missing),
```
Comment (chochem) [inline src/main/java/com/dreammaster/scripts/ScriptThaumicTinkerer.java:2094]: also looks like you removed 3 items by mistake

**Review by chochem — CHANGES_REQUESTED** [review]
see line comments

**Comment by Dream-Master** [comment]
@dagger8243

**Pull request closed without merging**